generators/entity/prompts.js

```diff
@@ -237,6 +238,45 @@ function askForRelationsToRemove() {
     }.bind(this));
 }
 
+function askForTableName() {
+    // don't prompt if there are no relationships
+    var entityTableName = this.entityTableName;
```

Comment:
If entityTableName is not specified in the entity.json, this will be undefined.  This causes the following if statement to fail.

Try something along the lines of
`    var entityTableName = this.entityTableName || this.getTableName(this.entityNameCapitalized);`